aiogram==2.9.2
https://github.com/aiogram/aiogram/issues/357

diff --git a/aiogram/types/message.py b/aiogram/types/message.py
index c56a143..2485fa5 100644
--- a/aiogram/types/message.py
+++ b/aiogram/types/message.py
@@ -1058,6 +1058,7 @@ class Message(base.TelegramObject):
         duration: typing.Union[base.Integer, None] = None,
         performer: typing.Union[base.String, None] = None,
         title: typing.Union[base.String, None] = None,
+        thumb: typing.Union[typing.Union[base.InputFile, base.String], None] = None,
         disable_notification: typing.Union[base.Boolean, None] = None,
         reply_markup: typing.Union[
             InlineKeyboardMarkup,
@@ -1107,6 +1108,7 @@ class Message(base.TelegramObject):
             duration=duration,
             performer=performer,
             title=title,
+            thumb=thumb,
             disable_notification=disable_notification,
             reply_to_message_id=self.message_id if reply else None,
             reply_markup=reply_markup,
